tsconfig.json

```diff
@@ -18,7 +21,7 @@
     "pretty": false,
     "resolveJsonModule": true,
     "sourceMap": true,
-    "target": "es6",
+    "target": "ES2020",
```

Comment:
Likewise, upping the target ES version was also required. They're definitely not concerned with backwards compatibility!